src/adaptation/AccessRule.cc

```diff
@@ -36,7 +36,7 @@ void
 Adaptation::AccessRule::finalize()
 {
     if (!group()) { // no explicit group
-        debugs(93,7, HERE << "no service group: " << groupId);
+        debugs(93,7, "no service group: " << groupId);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.